src/Documentation/2.0/New.md

```diff
@@ -19,4 +19,10 @@ The most significant changes in 2.0 are as follows.
 
 * Overhaul of code generation. While mostly invisible for developer, code generation became much more robust in handling serialization of various possible types. Special handling is required for F# assemblies. See [Code generation](Codegen.md) section for more details.
 
+* Created a `Microsoft.Orleans.Core.Abstractions` NuGet package and moved/refactored several types into it. Grain code would most likely need to reference only these abstractions, where as the silo host and client will reference more of Orleans packages. We plan to rev and do breaking changes to this package less frequently.
```

Comment:
"We plan to rev and do breaking changes to this package" -> "We plan to change this package"